src/components/login/Login.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState, useEffect } from 'react';
+import Button from '@material-ui/core/Button';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import TextField from '@material-ui/core/TextField';
+import Typography from '@material-ui/core/Typography';
+import Container from '@material-ui/core/Container';
+import Copyright from './Copyright';
+import useStyles from './style';
+import LogoImage from '../aa.png';
+import getUserLogin from '../API/getUserLogin';
+const Login = (props: any) => {
+  const classes = useStyles();
+  const [user, setLoginUser] = useState([]);
+  const [error, setError] = useState(false);
+  const [helperText, setHelperText] = useState('');
+  const [username, setUserName] = useState('');
+  const [password, setPassword] = useState('');
+  const [disabled, setDisabled] = useState(false);
+  const handleUserName = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setUserName(event.target.value);
+  }
+  const handlePassword = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const letterNumber:RegExp = /^[0-9a-zA-Z]+$/;
+    const eventPassword= event.target.value;
+    if (eventPassword.length >= 4 && eventPassword.match(letterNumber)) {
+      setError(false);
+      setHelperText('');
+      setPassword(event.target.value);
+      setDisabled(false);
+    }
+    else {
+      setError(true);
+      setHelperText('should be at least 4 char and contain char and number');
+      setDisabled(true);
+    }
+  }
+  useEffect(() => {
+    getUserLogin().then(res => {
+      setLoginUser(res);
+    });
+  });
```

Comment:
Pleases add an empty array here in the dependencies so that it run one time. 
This way it will run each time component rendered.